packages/client/src/arpa_reporter/views/UsersView.vue

```diff
@@ -4,7 +4,7 @@
     <div class="mb-4">
       <router-link
         to="/users/new"
-        class="btn btn-primary"
+        class="btn usdr-btn-primary"
```

Comment:
Note to reviewer:  before/after screenshots

## Before
![image](https://github.com/user-attachments/assets/dcd6c772-7c07-4adb-ad00-9de231f9467f)


## After
![image](https://github.com/user-attachments/assets/f28a2066-4ca3-4e25-b6ad-d6631ae65c28)